### ISSUE TITLE
Some accelerator keys are not available

### ISSUE THREAD

**Issue by martinhoess** [agustinmista/positron#13]
Some accelerator keys cannot be defined. for example: F12. (F12 is the only one I've found so far)
I think these are some standard shortcuts in Electron? (F12 would probably be Developer Tools? like in Chrome). 
But in combination with shift and others, it works. 

![image](https://user-images.githubusercontent.com/47973549/230998811-87cd0678-e355-4f37-bb77-6a2c7c878513.png)

**Comment by agustinmista** [comment]
Right, this seems to be a limitation of Electron registering some shortcuts for itself. Perhaps it can be disabled programmatically, I will look into that.

On a side note, I personally bypass these kinds of problems by simply remapping the keyboard keys to something fresh, like in the F13~F24 range. If you don't have a programmable keyboard, though, I think it can also be done using [PowerToys](https://learn.microsoft.com/en-us/windows/powertoys).
